datalake/src/main/java/com/sequenceiq/datalake/controller/sdx/SdxController.java

```diff
@@ -239,4 +248,19 @@ public FlowIdentifier stopByCrn(@ResourceCrn String crn) {
         return cdpConfigService.getAdvertisedRuntimes();
     }
 
+    // custom authorization check
+    @Override
+    @DisableCheckPermissions
+    public void setRangerCloudIdentityMapping(String envCrn, SetRangerCloudIdentityMappingRequest request) {
+        checkIsInternalActor();
```

Comment:
isn't already there a general solution for this @horadla23 ?

Reply:
yes, there is, last week we introduced `@InternalOnly` annotation, that should be used here instead of `@DisabledCheckPermissions` (I need to update authz readme)

Reply:
and `@InternalOnly` will guarantee that the method can be called only by internal actor

Reply:
Thanks, I've switched the new annotation.